pkg/plugins/common/kustomize/v2/scaffolds/internal/templates/config/crd/kustomization.go

```diff
@@ -110,7 +110,7 @@ var kustomizationTemplate = `# This kustomization.yaml is not intended to be run
 resources:
 %s
 
-patchesStrategicMerge:
+patches:
```

Comment:
@Sajiyah-Salat Try to make this change again in an up-to-date branch (upstream/master) and run `make generate`. After running `make generate` you will see that these files are also updated:
- docs/book/src/component-config-tutorial/testdata/project/config/crd/kustomization.yaml
- pkg/plugins/common/kustomize/v2/scaffolds/internal/templates/config/crd/kustomization.go
- testdata/project-v4-config/config/crd/kustomization.yaml
- testdata/project-v4-declarative-v1/config/crd/kustomization.yaml
- testdata/project-v4-multigroup/config/crd/kustomization.yaml
- testdata/project-v4-with-deploy-image/config/crd/kustomization.yaml
- testdata/project-v4/config/crd/kustomization.yaml

then commit the changes.